spec/shelter_spec.rb

```diff
@@ -3,44 +3,63 @@
 
 RSpec.describe Shelter do
 
+  before(:each) do
+
+  end
+
   # Iteration 1
   describe '#initialize' do
-    xit 'is a Shelter' do
-      shelter = Shelter.new('Denver Animal Shelter', 5)
+    it 'is a Shelter' do
+      shelter = Shelter.new('Denver Animal Shelter', 5,[])
+      
       expect(shelter).to be_a(Shelter)
+      # => #<Shelter:0x00000001048f0ed0 @capacity=5, @name="Denver Animal Shelter", @pets=[]>
     end
 
-    xit 'can read the name' do
+    it 'can read the name' do
       shelter = Shelter.new('Denver Animal Shelter', 5)
       expect(shelter.name).to eq('Denver Animal Shelter')
+      # => "Denver Animal Shelter"
     end
+    
 
-    xit 'can read the capacity' do
+    it 'can read the capacity' do
       shelter = Shelter.new('Denver Animal Shelter', 5)
+
       expect(shelter.capacity).to eq(5)
+      # => 5
     end
 
-    xit 'has no pets by default' do
+    it 'has no pets by default' do
       shelter = Shelter.new('Denver Animal Shelter', 5)
+
       expect(shelter.pets).to eq []
+      # => []
     end
   end
 
   # Iteration 2
   describe '#add_pet' do
-    xit 'returns a list of pets' do
-      shelter = Shelter.new('Denver Animal Shelter', 5)
+    it 'returns a list of pets' do
+            shelter = Shelter.new('Denver Animal Shelter', 5)
       shelter.add_pet('Salem')
+      # => ["Salem"]
       shelter.add_pet('Beethoven')
+      # => ["Salem", "Beethoven"]
       shelter.add_pet('Spot')
+      #=> ["Salem", "Beethoven", "Spot"]
       shelter.add_pet('Jonesy')
+      # => ["Salem", "Beethoven", "Spot", "Jonesy"]
 
       expect(shelter.pets).to eq(['Salem', 'Beethoven', 'Spot', 'Jonesy'])
+      # => ["Salem", "Beethoven", "Spot", "Jonesy"]
     end
   end 
 
   describe '#call_pets' do
-      xit 'returns a list of names with exclamation points appended' do
+
+    require 'pry'; binding.pry
```

Comment:
Because this pry is _outside_ of the `it` block, you don't have access here to any of the objects in this (or any other) test.  Pry is most useful when it is inside of a block of code - a test, a method, or even an enumerable.